assets/lokomotive-kubernetes/bootkube/resources/charts/lokomotive/templates/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: webhook-admission
+  labels:
+    name: webhook-admission
```

Comment:
There is no namespace information here.

Reply:
https://github.com/kinvolk/lokomotive/pull/704#discussion_r466904705